tsconfig.json

```diff
@@ -30,5 +36,13 @@
     ".next/types/**/*.ts",
     "data/clean.mts"
   ],
-  "exclude": ["node_modules"]
-}
+  "exclude": [
+    "node_modules"
+  ],
+  "ts-node": {
+    "experimentalSpecifierResolution": "node",
+    "compilerOptions": {
+      "module": "CommonJS",
```

Comment:
ah this is nice didn't realize you could set `ts-node` compiler options like this

Reply:
i should be able to remove the extra flag in the npm `prepare` script now

Reply:
Yes! Also a part of what I was exploring with the previous note. Removed the flag from the `package.json` as well.